crates/cast/tests/e2e/account/add.rs

```diff
@@ -88,15 +86,16 @@ pub async fn test_happy_case_add_profile() {
     ];
 
     let snapbox = Command::new(cargo_bin!("sncast"))
-        .current_dir(&current_dir)
+        .current_dir(&current_dir.path())
         .args(args);
 
     snapbox.assert().stdout_matches(indoc! {r#"
         command: account add
         add_profile: Profile successfully added to Scarb.toml
     "#});
-
-    let mut file = current_dir.clone();
+    let current_dir_utf8 =
+        Utf8PathBuf::from_path_buf(current_dir.into_path()).expect("Path contains invalid UTF-8"); // current_dir is consumed but the tmp folder is not deleted, it will persist until a reboot
```

Comment:
I appreciate it, but let's maybe obstain from unnecessary comments in the code :) 
```suggestion
        Utf8PathBuf::from_path_buf(current_dir.into_path()).expect("Path contains invalid UTF-8");
```